Styles/IHMhillsade.json

```diff
@@ -12,7 +12,7 @@
       "url": "https://israelhiking.osm.org.il/vector/data/TerrainRGB.json"
     }
   },
-  "sprite": "https://israelhikingmap.github.io/VectorMap/Icons/publish/sprite",
+  "sprite": "https://israelhiking.osm.org.il/content/sprite/sprite",
```

Comment:
I wonder why this raster-dem style has `sprite` and `glyphs`.

[MapLibre Style Spec](https://maplibre.org/maplibre-style-spec/sprite/) is not clear if they are a must .